src/hotspot/cpu/riscv/assembler_riscv.hpp

```diff
@@ -537,9 +537,9 @@ class Assembler : public AbstractAssembler {
   }
 
   INSN(lb,  0b0000011, 0b000);
-  INSN(lbu, 0b0000011, 0b100);
-  INSN(lh,  0b0000011, 0b001);
-  INSN(lhu, 0b0000011, 0b101);
+  INSN(_lbu, 0b0000011, 0b100); // Zcb
+  INSN(_lh,  0b0000011, 0b001); // Zcb
+  INSN(_lhu, 0b0000011, 0b101); // Zcb
```

Comment:
The code comment for these three lines seems misleading. These are normal 4-bytes encoding load/store instructions, not `Zcb` compressed instructions.

Reply:
The comment was suppose to mean, these are 'overridden' by zcb, not C as the others.
I'll clarify that.

Reply:
I remove them, fixed.